multipart-s3/src/utils/s3.rs

```diff
@@ -13,20 +12,42 @@ pub struct Client {
 impl Client {
     // construct S3 testing client
     pub fn new() -> Client {
-        let region = Region::ApNortheast2;
+      let env_region = std::env::var("AWS_REGION").unwrap();
+      let region = match env_region.as_ref() {
+        "ap-east-1" => Region::ApEast1,
+        "ap-northeast-1" => Region::ApNortheast1,
+        "ap-northeast-2" => Region::ApNortheast2,
+        "ap-south-1" => Region::ApSouth1,
+        "ap-southeast-1" => Region::ApSoutheast1,
+        "ap-southeast-2" => Region::ApSoutheast2,
+        "ca-central-1" => Region::CaCentral1,
+        "eu-central-1" => Region::EuCentral1,
+        "eu-north-1" => Region::EuNorth1,
+        "eu-west-1" => Region::EuWest1,
+        "eu-west-2" => Region::EuWest2,
+        "eu-west-3" => Region::EuWest3,
+        "me-south-1" => Region::MeSouth1,
+        "sa-east-1" => Region::SaEast1,
+        "us-east-1" => Region::UsEast1,
+        "us-east-2" => Region::UsEast2,
+        "us-west-1" => Region::UsWest1,
+        "us-west-2" => Region::UsWest2,
+        // Default
+        _ => Region::ApNortheast2
+      };
```

Comment:
We should use `Region::default()` here. ref: https://docs.rs/rusoto_core/0.43.0/rusoto_core/enum.Region.html#default

Reply:
so Thanks.I will fix it

Reply:
`Region::default()` will attempt to read `AWS_REGION` env so we don't need this match expression.

Reply:
So sorry to late fix.

Reply:
No worries, thanks for the work!